src/Draco.Compiler.Tests/Semantics/SymbolResolutionTests.cs

```diff
@@ -25,6 +33,20 @@ private static void AssertParentOf(Binder? parent, Binder? child)
         Assert.True(ReferenceEquals(childParent, parent));
     }
 
+    private static MetadataReference CompileCSharpToMetadataRef(string code)
+    {
+        var stringText = RoslynSourceText.From(code, Encoding.UTF8);
+        var tree = RoslynSyntaxFactory.ParseSyntaxTree(stringText);
+        var defaultReferences = Basic.Reference.Assemblies.Net70.ReferenceInfos.All.Select(r => RoslynMetadataReference.CreateFromStream(new MemoryStream(r.ImageBytes)));
+
+        var compilation = CSharpCompilation.Create("Test.dll", new RoslynSyntaxTree[] { tree }, defaultReferences, new CSharpCompilationOptions(Microsoft.CodeAnalysis.OutputKind.DynamicallyLinkedLibrary));
+        var stream = new MemoryStream();
+        var emitResult = compilation.Emit(stream);
+        Assert.True(emitResult.Success);
+        stream.Position = 0;
+        return MetadataReference.FromPeStream(stream);
+    }
```

Comment:
You could move this to a test-base class to have all this aliasing caused by it there, and for other tests for potential reuse